openmc/mgxs/mgxs.py

```diff
@@ -252,7 +252,7 @@ def __deepcopy__(self, memo):
             clone._name = self.name
             clone._rxn_type = self.rxn_type
             clone._by_nuclide = self.by_nuclide
-            clone._nuclides = copy.deepcopy(self._nuclides)
+            clone._nuclides = copy.deepcopy(self._nuclides, memo)
```

Comment:
I wasn't questioning your implementation, just was curious if that was the reasoning. I think it should probably remain as is.